<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hieuChinhDocGia_dialog</class>
 <widget class="QDialog" name="hieuChinhDocGia_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="maThe_lineEdit">
      <property name="cursor">
       <cursorShape>BusyCursor</cursorShape>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="hoVaTen_label">
      <property name="text">
       <string>Họ và Tên</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="hoVaTen_lineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="gioiTinh_comboBox">
      <item>
       <property name="text">
        <string>Nam</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nữ</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="gioiTinh_label">
      <property name="text">
       <string>Giới Tính </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="maThe_label">
      <property name="text">
       <string>Mã thẻ</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="trangThaiThe_label">
      <property name="text">
       <string>Trạng thái thẻ</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="trangThaiThe_comboBox">
      <item>
       <property name="text">
        <string>Đang hoạt động</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Khóa</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Ok_pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
